source/proof-of-concept-guide/poc-vulnerability-detection.rst

```diff
@@ -51,12 +50,21 @@ The Vulnerability Detection module is enabled by default. You can perform the fo
 
       $ sudo systemctl restart wazuh-manager
 
+
+Test the configuration
+----------------------
+
+You don’t have to perform any action. It performs vulnerability detection scans periodically for applications and operating systems on each monitored endpoint. In this example we use the installed packages on the Ubuntu endpoint.
```

Comment:
```suggestion
You don’t have to perform any action. Wazuh detects the vulnerable packages installed in the Ubuntu endpoint automatically. The time it takes to detect vulnerabilities depends on the Syscollector update interval configured in the ``ossec.conf`` file.
```

Reply:
Consider adding a link to the interval syscolletor option referred.

Reply:
Done!